examples/websocket_client/WebSocketWrapper.cpp

```diff
@@ -6,7 +6,7 @@ using namespace rtcdcpp;
 
 WebSocketWrapper::WebSocketWrapper(std::string url) : url(url), send_queue() { ; }
 
-WebSocketWrapper::~WebSocketWrapper() { ; }
+WebSocketWrapper::~WebSocketWrapper() { delete this->ws; }
```

Comment:
I hadn't done things like this in the test because I was lazy, thanks for the cleanup